src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -401,6 +407,30 @@ private void AddCredentialToCache(Uri serverUri, ulong authAvail, NetworkCredent
             }
         }
 
+        private void AddResponseCookies(Uri serverUri, HttpResponseMessage response)
+        {
+            if (!_useCookie)
```

Comment:
Thanks.  I missed that the HttpClientHandler's ctor was overriding the default set by the WinHttpHandler's ctor.